modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldDefaultValue.testcase

```diff
@@ -209,12 +297,131 @@ definition {
 		}
 	}
 
-	@description = "LPS-163716 - Verify updating the default value configuration for the Picklist field affects the object entries"
+	@description = "LPS-181204 - Verify updating the default value configuration for the Picklist field affects the object entries"
```

Comment:
Verify updating the default value configuration for the Picklist field only affects new object entries

Reply:
@victorhcunha can you also update the test name to `CanOnlyAffectNewEntriesByUpdatingConfiguration`